.changeset/early-lions-turn.md

```diff
@@ -0,0 +1,21 @@
+---
+'@leafygreen-ui/code': major
```

Comment:
Should we add a separate changeset for the Syntax component indicating that it's deprecated?

Reply:
Decided not to do this. I don't believe anyone's using the package, and if they are, the current version is on NPM.

Reply:
That's fine, then let's do the same for MongoMenu

Reply:
I think our approach to MongoMenu was correct, and was a different case, but it was never meant to stick around as long as it did. I'm alright with removing it at this point.

Reply:
*In a separate PR

Reply:
sgtm let's just document that this change needs to be made

Reply:
https://jira.mongodb.org/browse/PD-1012

Reply:
Thanks!